packages/shared-components/src/Leaderboard.tsx

```diff
@@ -81,59 +88,59 @@ function Leaderboard({
-                rank={entry.rank}
-                pseudonym={entry.username}
-                avatar={entry.avatar ?? 'placeholder'}
-                withAvatar={!hideAvatars}
-                points={entry.score}
-                onClick={
-                  onParticipantClick
-                    ? () => onParticipantClick(entry.participantId, false)
-                    : undefined
-                }
-                className={className?.listItem}
-              />
-            )
-          )}
-        </div>
+        {activeParticipation && (
+          <div className={twMerge('space-y-1', className?.list)}>
+            {top10.map((entry: LeaderboardEntry) =>
+              entry.isSelf === true && onLeave ? (
+                <ParticipantSelf
+                  key={entry.id}
+                  isActive={activeParticipation ?? true}
+                  pseudonym={entry.username}
+                  level={entry.level}
+                  avatar={entry.avatar}
+                  withAvatar={!hideAvatars}
+                  points={entry.score}
+                  rank={entry.rank}
+                  onJoinCourse={onJoin}
+                  onLeaveCourse={onLeave}
+                  onClick={
+                    onParticipantClick
+                      ? () => onParticipantClick(entry.participantId, true)
+                      : undefined
+                  }
+                />
+              ) : (
+                <ParticipantOther
+                  key={entry.id}
+                  rank={entry.rank}
+                  pseudonym={entry.username}
+                  level={entry.level}
+                  avatar={entry.avatar}
+                  withAvatar={!hideAvatars}
+                  points={entry.score}
+                  onClick={
+                    onParticipantClick
+                      ? () => onParticipantClick(entry.participantId, false)
+                      : undefined
+                  }
+                  className={className?.listItem}
+                />
+              )
+            )}
+          </div>
+        )}
         {participant?.id && onJoin && onLeave && !activeParticipation && (
-          <ParticipantSelf
-            isActive={activeParticipation ?? false}
-            pseudonym={participant.username}
-            avatar={participant.avatar ?? 'placeholder'}
-            withAvatar={!hideAvatars}
-            points={selfEntry?.score}
-            rank={selfEntry?.rank ?? '?'}
-            onJoinCourse={onJoin}
-            onLeaveCourse={onLeave}
-            onClick={
-              onParticipantClick
-                ? () => onParticipantClick(participant.id, true)
-                : undefined
-            }
-          />
+          <div className="max-w-none p-2 bg-slate-100 rounded border-slate-300 border text-slate-600 text-sm">
+            <Markdown
+              withProse
+              withLinkButtons={false}
+              content={t('pwa.general.joinLeaderboardNotice')}
```

Comment:
still being written, will be added to de.json and en.json once done, untranslated string for now